.github/workflows/populate-cache.yml

```diff
@@ -0,0 +1,34 @@
+# A dummy workflow that populates the runner with dependencies
+# and pops them in the actions cache, so we know there is
+# always a fallback for subsequent PRs
+#
+name: Populate Runner Cache
+on:
+  push:
+    branches:
+      - main
+      - feature/**
+  pull_request:
+    branches-ignore:
+      - docs
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  cache:
+    name: ${{ matrix.config.os }}
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        # config.os vs os to match requirements of bootstrap_platform
+        config:
+          - os: ubuntu-20.04
+          - os: macos-11
+          - os: windows-2019
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Dependabot just updated this to v3, should likely follow suit.

Reply:
Hahah totally yeah - good catch thanks!